ui/src/organizations/components/CollectorRow.tsx

```diff
@@ -30,6 +31,7 @@ export default class BucketRow extends PureComponent<Props> {
               size={ComponentSize.Small}
               color={ComponentColor.Secondary}
               text={'Download Config'}
+              onClick={this.handleDownloadConfig}
```

Comment:
Can you combine this column and the last? So `Download` is in the same column as `Delete`?
You can wrap the buttons in a clockface `ComponentSpacer` to ensure there is a little gap between the two. See the action buttons in the dashboards list for an example of how to do this

Reply:
Fixed it! Thanks :)